src/views/ThreeWActivityForm/schema.ts

```diff
@@ -0,0 +1,1279 @@
+            status: {},
+            sectors: {
+                forceValue: undefinedValue,
+                validations: [countGreaterThanLength(sectorsInActivities?.length ?? 0)],
+            },
+        };
+
+        schema = addCondition(
+            schema,
+            value,
+            ['start_date', 'end_date'] as const,
+            ['end_date'] as const,
+            (props): Pick<FormSchemaFields, 'end_date'> => {
+                if (props?.start_date) {
+                    return {
+                        end_date: {
+                            required: true,
+                            validations: [dateGreaterThanOrEqualCondition(props.start_date)],
+                        },
+                    };
+                }
+                return {
+                    end_date: { required: true },
+                };
+            },
+        );
+
+        const reportingNsFields = [
+            'reporting_ns',
+            'reporting_ns_contact_name',
+            'reporting_ns_contact_email',
+            'reporting_ns_contact_role',
+            'deployed_eru',
+        ] as const;
+
+        type ReportingNsSchema = Pick<FormSchemaFields, (typeof reportingNsFields)[number]>;
+
+        schema = addCondition(
+            schema,
+            value,
+            ['activity_lead'] as const,
+            reportingNsFields,
+            (props): ReportingNsSchema => {
+                if (props?.activity_lead === 'national_society') {
+                    return {
+                        reporting_ns: { required: true },
+                        reporting_ns_contact_name: { required: true },
+                        reporting_ns_contact_email: {
+                            required: true,
+                            validations: [emailCondition],
+                        },
+                        reporting_ns_contact_role: { required: true },
+                        deployed_eru: { forceValue: undefinedValue },
+                    };
+                }
+                if (props?.activity_lead === 'deployed_eru') {
+                    return {
+                        reporting_ns: { forceValue: undefinedValue },
+                        reporting_ns_contact_name: { forceValue: undefinedValue },
+                        reporting_ns_contact_role: { forceValue: undefinedValue },
+                        reporting_ns_contact_email: { forceValue: undefinedValue },
+                        deployed_eru: { required: true },
+                    };
+                }
+                return {
+                    reporting_ns: { forceValue: undefinedValue },
+                    deployed_eru: { forceValue: undefinedValue },
+                    reporting_ns_contact_name: { forceValue: undefinedValue },
+                    reporting_ns_contact_role: { forceValue: undefinedValue },
+                    reporting_ns_contact_email: { forceValue: undefinedValue },
+                };
```

Comment:
A fool proof method to do this is first create "forceNull" fields
Then update them to have values on each branch..
Also why are we using `forceValue: undefinedValue`